conwayste/src/ui/pane.rs

```diff
@@ -57,7 +62,28 @@ impl Pane {
             previous_pos: None,
             border: 1.0,
             bg_color: None,
-        }
+            handlers: Some(context::HandlerMap::new()),
+        };
+        // TODO: propagate events for other EventTypes
+        let handler: context::Handler = Box::new(|obj, uictx, evt| {
```

Comment:
I think this would also benefit from a macro since it's nearly identical to the one in `Layer`.

Reply:
Done! I did something a little more abstract (set a handler for each event type that's a mouse event). See `forward_mouse_events!` macro.

I plan to do something similar for keyboard (which would use the focused widget not the widget under the cursor).